### ISSUE TITLE
Please implement in a way that provides context information in an Editor.

### ISSUE THREAD

**Issue by MarcSkovMadsen** [paulopes/panel-components#2]
I've been sitting here trying to wrap my head around how to use this. I simply do not understand how to get started.

#### Solution

- Please provide some examples (or example links) and screenshots in the README.
- Please refactor the implementation so that static analysis can be done and help context help provided in editors

![image](https://user-images.githubusercontent.com/42288570/94641744-5d882500-02e2-11eb-88e8-da280bc71893.png)

But unfortunately this means that I get no context help/ intellisense when working in an editor like VS Code.

One way to enable static analysis would be to auto generate code strings with docstrings from the list of tags.

**Comment by MarcSkovMadsen** [comment]
(Now I do know how to get started. See other issues and PRs).

**Comment by paulopes** [comment]
Marc,

I’m incredibly busy this week, and next week as well.

I’ll try to address your questions and requests as I find some time to look at them.

Kind regards,

  Paulo Lopes


On Sep 30, 2020, at 12:03 AM, Marc Skov Madsen <notifications@github.com> wrote:


I've been sitting here trying to wrap my head around how to use this. I simply do not understand how to get started.

Solution

Please provide some examples (or example links) and screenshots in the README.
Please refactor the implementation so that static analysis can be done and help context help provided in editors
 <https://user-images.githubusercontent.com/42288570/94641744-5d882500-02e2-11eb-88e8-da280bc71893.png>
But unfortunately this means that I get now context help when working in an editor like VS Code.

One way to enable static analysis would be to auto generate code strings with docstrings from the list of tags.

**Comment by MarcSkovMadsen** [comment]
No worries @paulopes . I will just record issues and do a few PRs in order to “record” the experience I have, such that it can be used when the time and energy is there.

**Comment by MarcSkovMadsen** [comment]
Docstrings would be nice as well :-)

**Comment by paulopes** [comment]
I agree, I just can’t do any of these things right now, or even respond to most of your messages, sorry. I’m too busy.


On Sep 30, 2020, at 2:47 PM, Marc Skov Madsen <notifications@github.com> wrote:


Docstrings would be nice as well :-)

**Comment by paulopes** [comment]
Thanks :-)

On Sep 30, 2020, at 12:18 PM, Marc Skov Madsen <notifications@github.com> wrote:


No worries @paulopes <https://github.com/paulopes> . I will just record issues and do a few PRs in order to “record” the experience I have, such that it can be used when the time and energy is there.
